toonz/sources/translations/german/colorfx.ts

```diff
@@ -384,8 +384,8 @@
 <context>
     <name>TLinGradFillStyle</name>
     <message>
-        <source>Linear Gradient</source>
-        <translation>Linearer Gradient</translation>
+        <source>Linear Verlauf</source>
```

Comment:
Are you sure the source should be changed?
Also I wouldn't translate "Gradient" to "Verlauf" (see https://de.wikipedia.org/wiki/Gradient_%28Grafik%29)